posts/2020-11-26-MP-rest-client-Kubernetes-secrets-21001.adoc

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+title: "MP Rest Client 2.0, using Kubernetes secrets in Liberty variables and more Jakarta EE 9 features in Open Liberty 21.0.0.1-beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: MP Rest Client 2.0, using Kubernetes secrets in Liberty variables and more Jakarta EE 9 features in Open Liberty 21.0.0.1-beta - OpenLiberty.io
+seo-description: MP Rest Client 2.0, using Kubernetes secrets in Liberty variables. Plus Jakarta Faces Container 3.0 and Jakarta Enterprise Beans Persistent Timer 4.0 included in Jakarta EE 9
+blog_description: MP Rest Client 2.0, using Kubernetes secrets in Liberty variables. Plus Jakarta Faces Container 3.0 and Jakarta Enterprise Beans Persistent Timer 4.0 included in Jakarta EE 9
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
```

Comment:
Change this to point at one of the images in this blog, like https://openliberty.io/img/blog/blog_21001_deployment_definition.png

Reply:
Done